org.cvfosammmm.Lemma.yml

```diff
@@ -0,0 +1,49 @@
+id: org.cvfosammmm.Lemma
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: lemma
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=host
```

Comment:
why host?

Reply:
It's currently not needed. What would you recommend?

Reply:
removed